### ISSUE TITLE
Generate other pages from the DB

### ISSUE THREAD

**Issue by pauloney** [TeXUsersGroup/TUG-Conference#6]
Make a database of `speakers` and generate the main-page of the site automatically from it, in the same fashion of make-schedule.js, that is, using a template.

**Comment by kberry** [comment]
My idea has been: people contains author's names and email addresses, etc.
tlk.dat contains per-presentation info (title, list of authors, etc.)
auth.dat contains per-author name when needed to link between people and tlk.dat, specifically, when the names don't match.

The idea of this was to have *one* definitive place for each piece of information. It's a slippery slope to do anything else.

**Comment by pauloney** [comment]
Yes, that is the correct way and that is called a "relational" database,
and that is what I wanted to do from day one...but now we will have to talk
to Arthur and change things a bit.

Arthur last year made the first implementation of the program to write my
html code for the program. In there he used only ONE table listing all
presentations and as many authors as needed. So Vafa Khalighi's name is
listed there 3 times....along with others that gave multiple lectures.

The is also no way to associate email to people unless we use order which
is cumbersome...let alone multiple email to the same person, etc...

This table contains all the abstracts as well...in html and as one liner.

Paulo  Ney
